src/components/Referrals/referralsHelper.js

```diff
@@ -10,10 +10,17 @@ import { ARBITRUM, AVALANCHE } from "config/chains";
 import { bigNumberify, formatAmount } from "lib/numbers";
 import { t } from "@lingui/macro";
 import { getRootUrl } from "lib/url";
+import { BASIS_POINTS_DIVISOR } from "config/factors";
 
 export const REFERRAL_CODE_REGEX = /^\w+$/; // only number, string and underscore is allowed
 export const REGEX_VERIFY_BYTES32 = /^0x[0-9a-f]{64}$/;
 
+export function removeTrailingZeros(amount) {
```

Comment:
probably it's better to move this function to lib/numbers

Reply:
Updated